### PR TITLE
AArch64: Implement vdiv evaluator for integer elements

### PR DESCRIPTION
**Pull request by Akira1Saitoh**
Implement vdiv evaluator for integer elements.

Signed-off-by: Akira Saitoh <saiaki@jp.ibm.com>

### CODE REVIEW

File: compiler/aarch64/codegen/BinaryEvaluator.cpp [hunk 1]
```diff
@@ -703,6 +703,144 @@ OMR::ARM64::TreeEvaluator::vmulEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    return inlineVectorBinaryOp(node, cg, mulOp);
    }
 
+TR::Register *
+OMR::ARM64::TreeEvaluator::vdivIntHelper(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   struct DivOps {
```
Comment (knn-k) [inline compiler/aarch64/codegen/BinaryEvaluator.cpp:709]: Please use the following indentation style:

```
   struct DivOps
      {
      TR::InstOpCode::Mnemonic vectorToGPROp;
      ...
      }
```

File: compiler/aarch64/codegen/BinaryEvaluator.cpp [hunk 2]
```diff
@@ -703,6 +703,144 @@ OMR::ARM64::TreeEvaluator::vmulEvaluator(TR::Node *node, TR::CodeGenerator *cg)
...
+   * sdivw  w0, w1, w0
+   * vinswb v2.b[11], w0
+   * smovwb w0, v0.b[12]
+   * smovwb w1, v1.b[12]
+   * sdivw  w0, w1, w0
+   * vinswb v2.b[15], w0
+   * vushr4s v0, v0, #8
+   * vushr4s v1, v1, #8
+   * subw  w2, w2, #1
+   * cbnzw w2, loop
```
Comment (knn-k) [inline compiler/aarch64/codegen/BinaryEvaluator.cpp:765]: I would suggest aligning the operands, or always using a single space between the mnemonic and the operands:

```
   * vushr4s v1, v1, #8
   * subw    w2, w2, #1
   * cbnzw   w2, loop
```

```
   * vushr4s v1, v1, #8
   * subw w2, w2, #1
   * cbnzw w2, loop
```

File: compiler/aarch64/codegen/BinaryEvaluator.cpp (hunk 1 above)
Comment (Akira1Saitoh) [reply compiler/aarch64/codegen/BinaryEvaluator.cpp]: Fixed the indentation.

File: compiler/aarch64/codegen/BinaryEvaluator.cpp (hunk 2 above)
Comment (Akira1Saitoh) [reply compiler/aarch64/codegen/BinaryEvaluator.cpp]: Fixed the alignment.

**Comment by knn-k** [comment]
Jenkins build aarch64

**Review by knn-k — APPROVED** [review]

**Pull request merged**